server/src/test/java/org/elasticsearch/indices/analysis/AnalysisModuleTests.java

```diff
@@ -240,34 +240,15 @@ public void testUnderscoreInAnalyzerName() throws IOException {
         }
     }
 
-    @AwaitsFix(bugUrl="https://github.com/elastic/elasticsearch/issues/38635")
     public void testStandardFilterBWC() throws IOException {
-        Version version = VersionUtils.randomVersionBetween(random(), Version.V_6_0_0, Version.CURRENT.minimumCompatibilityVersion());
-        // bwc deprecation
-        {
-            Settings settings = Settings.builder()
-                .put("index.analysis.analyzer.my_standard.tokenizer", "standard")
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_7_0_0, Version.CURRENT);
+        final Settings settings = Settings.builder().put("index.analysis.analyzer.my_standard.tokenizer", "standard")
```

Comment:
The standard tokenizer is still there, I think?  You need to add `index.analysis.analyzer.my_standard.filter=standard`

Reply:
Oh, never mind, it's still there - ignore me.